manifests/init.pp

```diff
@@ -303,6 +317,13 @@
   Enum['none', 'plain', 'login', 'cram-md5'] $email_smtp_authentication = $::foreman::params::email_smtp_authentication,
   Optional[String] $email_smtp_user_name = $::foreman::params::email_smtp_user_name,
   Optional[String] $email_smtp_password = $::foreman::params::email_smtp_password,
+  Optional[String] $telemetry_prefix = $::foreman::params::telemetry_prefix,
+  Boolean $telemetry_prometheus_enabled = $::foreman::params::telemetry_prometheus_enabled,
+  Boolean $telemetry_statsd_enabled = $::foreman::params::telemetry_statsd_enabled,
+  Optional[String] $telemetry_statsd_host = $::foreman::params::telemetry_statsd_host
+  Optional[Enum['statsd', 'statsite', 'datadog']] $telemetry_statsd_protocol = $::foreman::params::telemetry_statsd_protocol
+  Boolean $telemetry_logger_enabled = $::foreman::params::telemetry_logger_enabled
+  Optional[Enum['DEBUG', 'INFO', 'WARN', 'ERROR', 'FATAL']] $telemetry_logger_level = $::foreman::params::telemetry_logger_level
```

Comment:
This was meant to be for development, maybe at least mention this in docs comment. No practical use.

Reply:
the same as above, if we don't expose and people override, it would get reset every time installer runs